app/lib/server/functions/notifications/email.js

```diff
@@ -94,6 +94,13 @@ function getEmailContent({ message, user, room }) {
 	return header;
 }
 
+const getButtonUrl = (room, subscription, message) => {
+	const siteUrl = settings.get('Site_Url').replace(/https?\:\/\//i, '');
+	const host = s.rtrim(siteUrl, '/');
+	const path = `${ s.ltrim(roomTypes.getRelativePath(room.t, subscription), '/') }&msg=${ message._id }`;
+	return `${ settings.get('Offline_Message_Link_To_Redirect') }/room?host=${ host }&path=${ path }`;
```

Comment:
@MarcosSpessatto can we reuse the [existing getUrl](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/utils/lib/getURL.js#L21) method? And maybe add a setting just to decide whenever use the cloud url or the local url?